datasets/per_sent/per_sent.py

```diff
@@ -0,0 +1,145 @@
+
+PerSenT is a crowd-sourced dataset that captures the sentiment of an author towards the main entity in a news article. This dataset contains annotation for 5.3k documents and 38k paragraphs covering 3.2k unique entities.
+
+"""
+
+from __future__ import absolute_import, division, print_function
+
+import csv
+
+import datasets
+from datasets.splits import NamedSplit
+
+
+# TODO: Add BibTeX citation
+# Find for instance the citation on arxiv or on the dataset repo/website
+_CITATION = """\
+@inproceedings{bastan2020authors,
+      title={Author's Sentiment Prediction},
+      author={Mohaddeseh Bastan and Mahnaz Koupaee and Youngseo Son and Richard Sicoli and Niranjan Balasubramanian},
+      year={2020},
+      eprint={2011.06128},
+      archivePrefix={arXiv},
+      primaryClass={cs.CL}
+}
+"""
+
+_DESCRIPTION = """\
+Person SenTiment (PerSenT) is a crowd-sourced dataset that captures the sentiment of an author towards the main entity in a news article. This dataset contains annotation for 5.3k documents and 38k paragraphs covering 3.2k unique entities.
+
+The dataset consists of sentiment annotations on news articles about people. For each article, annotators judge what the author’s sentiment is towards the main (target) entity of the article. The annotations also include similar judgments on paragraphs within the article.
+
+To split the dataset, entities into 4 mutually exclusive sets. Due to the nature of news collections, some entities tend to dominate the collection. In the collection, there were four entities which were the main entity in nearly 800 articles. To avoid these entities from dominating the train or test splits, we moved them to a separate test collection. We split the remaining into a training, dev, and test sets at random. Thus our collection includes one standard test set consisting of articles drawn at random (Test Standard -- `test_random`), while the other is a test set which contains multiple articles about a small number of popular entities (Test Frequent -- `test_fixed`).
+"""
+
+_LICENSE = "Creative Commons Attribution 4.0 International License"
+
+_URLs = {
+    "train": "https://raw.githubusercontent.com/MHDBST/PerSenT/main/train.csv",
+    "dev": "https://raw.githubusercontent.com/MHDBST/PerSenT/main/dev.csv",
+    "test_random": "https://raw.githubusercontent.com/MHDBST/PerSenT/main/random_test.csv",
+    "test_fixed": "https://raw.githubusercontent.com/MHDBST/PerSenT/main/fixed_test.csv",
+}
+
+
+class PerSenT(datasets.GeneratorBasedBuilder):
+    """Person SenTiment (PerSenT) is a crowd-sourced dataset that captures the sentiment of an author towards the main entity in a news article. This dataset contains annotation for 5.3k documents and 38k paragraphs covering 3.2k unique entities."""
+
+    VERSION = datasets.Version("1.1.0")
+
+    def _info(self):
+        label = datasets.features.ClassLabel(names=["", "Negative", "Neutral", "Positive"])
```

Comment:
```suggestion
        label = datasets.features.ClassLabel(names=["Negative", "Neutral", "Positive"])
```
Usually we only use the valid class label names here, and when a label is missing we use the value `-1` in the example dictionary that is yielded

Could you change that ?
You'll need to re-generate the dataset_infos.json after this change